lib/memento/strategy/ram.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Memento.Strategy.RAM do
+  @moduledoc """
+  Module for using Memento in a RAM-based configuration
+
+  `Memento.Strategy.RAM` starts Memento in a RAM-only
+  configuration. Each node joined to the cluster attempts
+  to create the desired tables on startup, but only the
+  first succeeds. The others detect the existing tables
+  on failure and then add them from remote node(s).
+
+  Unsplit is handled in a decentralized manner. In the
+  event of a netsplit, the minority node(s) are aware of
+  their situation and refuse to write. They attempt rejoin
+  to the majority on the write failures.
+  """
+
+  @behaviour Memento.Strategy
+  require Memento.Mnesia
+
+  @wait_timeout 60_000
+
+  @impl true
+  def startup(tables, nodes \\ Node.list()) do
+    Memento.start()
```

Comment:
This line should be moved to `Memento.Supervisor.init/1` before calling the strategy's startup callback, no? Or is the goal here to let the strategy author do any extra tasks they wish to do before memento/mnesia starts?